### ISSUE TITLE
Missing initWithReuseIdentifier constructor in FZAccordionTableViewHeaderView

### ISSUE THREAD

**Issue by borut-t** [fuzz-productions/FZAccordionTableView#5]
FZAccordionTableViewHeaderView should implement the following constructor when header is not initialised with nib.

```
- (id)initWithReuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithReuseIdentifier:reuseIdentifier];
    if (self) {
        [self addGestureRecognizer:[[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(touchedHeaderView:)]];
    }
    return self;
}
```

**Comment by kgaidis** [comment]
Thank you! I'll look into this as soon as possible.

**Comment by kgaidis** [comment]
Should be fixed now with release 0.1.3 !
